### PR TITLE
54 Disables airjump and dash while human

### PR DESCRIPTION
**Pull request by MrJackey**
## Summary
Fixes # 54
Toggles a variable depending on if transforming to human.
This disables air jumps and dash.

### CODE REVIEW

File: Assets/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -61,6 +61,10 @@ public class PlayerController : MonoBehaviour {
 	private float dashTimer = 0f;
 	private float dashResetTimer = 0.5f;
 
+	public bool HumanControls {
+		set => humanControls = value;
+	}
+
 	public bool AllowControls { set => allowControls = value; }
```
Comment (smeas) [inline Assets/Scripts/Player/PlayerController.cs]: Inconsistent formatting of properties.

**Review by smeas — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (MrJackey) [reply Assets/Scripts/Player/PlayerController.cs]: True, will fix

**Review by smeas — APPROVED** [review]
+1, Great!

**Review by Addehman — APPROVED** [review]
Ser gytt!

**Review by smeas — APPROVED** [review]

**Pull request merged**